### PR TITLE
Small refactor before adding ``unknown-message-in-pylintrc`` warning

### PR DESCRIPTION
**Pull request by Pierre-Sassoulas**
Before working on the unknown message in pylintrc disable/enable, some fix came to mind.

|     | Type                   |
| --- | ---------------------- |
| ✓   | :hammer: Refactoring   |
| ✓   | :scroll: Docs          |

## Related Issue

Inspired by #4265

### CODE REVIEW

File: pylint/config/option_manager_mixin.py [hunk 1]
```diff
@@ -236,22 +236,24 @@ def load_provider_defaults(self):
...
 
     def read_config_file(self, config_file=None, verbose=None):
-        """read the configuration file but do not load it (i.e. dispatching
-        values to each options provider)
-        """
-        helplevel = 1
-        while helplevel <= self._maxlevel:
-            opt = "-".join(["long"] * helplevel) + "-help"
+        """Read the configuration file but do not load it (i.e. dispatching
+        values to each options provider)"""
```
Comment (cdce8p) [inline pylint/config/option_manager_mixin.py:240]:
```suggestion
        """Read the configuration file but do not load it (i.e. dispatching
        values to each options provider)
        """
```
For multiline comments it's usually common practice to put the closing quotes on a new line.

Comment (cdce8p) [reply pylint/config/option_manager_mixin.py]: Just FWI, this wasn't the only related change with this issue :)

Comment (Pierre-Sassoulas) [reply pylint/config/option_manager_mixin.py]: I think PcManticore prefer the other way to do it that's why I do it like this. I actually like the version that take less lines too. I wish black would automatically do that. Also I think we should let those the way they are and not bother about it.

Comment (cdce8p) [reply pylint/config/option_manager_mixin.py]: I would imaging black would choose the PEP 8 [style](https://www.python.org/dev/peps/pep-0008/#documentation-strings) though, but I agree it's not worth arguing about small things like that.

Comment (Pierre-Sassoulas) [reply pylint/config/option_manager_mixin.py]: Ho if it's pep8 then I'm going to do it the other way next time, thanks for pointing that out.

**Pull request merged**